### PR TITLE
Fixing ring of recoil dmg, should hit 0s. Removed some @params issues…

### PR DESCRIPTION
**Pull request by syuil**
… with intellij -- wouldn't let me merge without removing them -- called them errors for some reason..

### CODE REVIEW

File: ElvargServer/src/main/java/com/elvarg/game/content/combat/CombatFactory.java [hunk 1]
```diff
@@ -824,7 +821,11 @@ public static void handleRecoil(Player player, Mobile attacker, int damage) {
 		if (damage == 0) {
 			return;
 		}
-		final int returnDmg = (int) (damage * 0.1) + 1;
+		int returnDmg = (int) (damage * 0.1) + 1;
+
+		if(returnDmg < 3 && new RandomGen().inclusive(1, 3) == 2) {
```
Comment (tobywisener) [inline ElvargServer/src/main/java/com/elvarg/game/content/combat/CombatFactory.java:826]: Do we need to create new instance of `RandomGen` on every hit to keep the numbers truly random? Probably not a big issue just wondering :)

Comment (syuil) [reply ElvargServer/src/main/java/com/elvarg/game/content/combat/CombatFactory.java:826]: I wasn't even paying attention, I just copy and pasted from the original source that was lost due to a merge.. Agree tho, shouldn't we just use misc.random?

Comment (tobywisener) [reply ElvargServer/src/main/java/com/elvarg/game/content/combat/CombatFactory.java:826]: Yeah, give it a try with Misc.random, at least it'll be a bit more consolidated 🗡️

**Review by tobywisener — APPROVED** [review]

**Comment by syuil** [comment]
Uploading a new PR soon.

**Pull request closed without merging**